### PR TITLE
feat: setup brevo for the reminder emails

### PR DESCRIPTION
**Pull request by lisbethpurrucker**
Set up Brevo SMTP for the reminder emails due to network issues when sending them through our usual Gmail smtp (too many login attempts).

### Checklist

- [x] Code compiles correctly
- [x] Created tests which fail without the change (if possible)
- [x] All tests passing
- [x] Rubocop passes

### CODE REVIEW

File: app/jobs/bookings_job.rb [hunk 1]
```diff
@@ -6,7 +7,13 @@ class BookingsJob < ApplicationJob
                                                                                         5.seconds * (2**attempt)
                                                                                       }, attempts: 10
```
Comment (papotte) [inline app/jobs/bookings_job.rb:9]: Shouldn't this be initialised with an initialiser in config?

**Review by papotte — COMMENTED** [review]
I'm missing the credentials for the Dev environment. Are we supposed to be able to use Brevo in localhost?

File: app/jobs/bookings_job.rb (hunk 1 above)
Comment (lisbethpurrucker) [reply app/jobs/bookings_job.rb]: I actually removed it entirely as it was my attempt to fix the `too many login attempts` error.

**Review by papotte — APPROVED** [review]

File: app/jobs/bookings_job.rb (hunk 1 above)
Comment (papotte) [reply app/jobs/bookings_job.rb]: We'll probably need a brevo initializer at some point if we're going to use it for other emails, but for now this is okay :D

Comment (lisbethpurrucker) [reply app/jobs/bookings_job.rb]: Oh, is that what you meant? 🤔

**Pull request merged**